packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -423,30 +424,44 @@ class Checkout extends Component<
     }
 
     private renderShippingStep(step: CheckoutStepStatus): ReactNode {
-        const { hasCartChanged, cart, consignments = [] } = this.props;
+        const { hasCartChanged, cart, consignments = [], isNewMultiShippingUIEnabled } = this.props;
 
         const { isBillingSameAsShipping, isMultiShippingMode } = this.state;
 
         if (!cart) {
             return;
         }
 
+        const isRenderMultiStaticConsignment = isNewMultiShippingUIEnabled && consignments.length > 1;
+
+        const summary = isRenderMultiStaticConsignment
```

Comment:
Can we put this into a separate component file as `checkout.tsx` is quite big now?